pykokkos/interface/views.py

```diff
@@ -417,7 +449,16 @@ def from_numpy(array: np.ndarray, space: Optional[MemorySpace] = None, layout: O
     if layout is None:
         layout = Layout.LayoutDefault
 
-    return View(list(array.shape), dtype, space=space, trait=Trait.Unmanaged, array=array, layout=layout)
+    # TODO: pykokkos support for 0-D arrays?
+    # temporary/terrible hack here for array API testing..
```

Comment:
So, the `pykokkos` tests proper are passing in this PR now. Of course, even the small subset of array API tests I've added in CI don't pass yet. There appear to be two main challenges:

1. Lack of native support for `bool` in Kokkos proper? I asked about this on the main Slack channel and there didn't seem to be any suggestions.
2. What we currently have exposed for 0-d array support. There were some comments on Slack which I'll just paste below:

> Kokkos has rank-0 View<T> which contains a single scalar value. It doesn’t have a unit-like View<void> or whatever

> yeah a rank-0 view is also accessible as you suggested by ()

```
View<int> a("A");
a() = 5;
```

> mdspan is the same (just with [] operator instead)

```
mdspan<int, dextents<0>> a(ptr_a);
a[] = 5;
```